lib/Utilities.ts

```diff
@@ -282,3 +282,40 @@ export function escapeDiscordMarkdown(text: string) {
         },
     );
 }
+
+export async function handleFloorPriceChannel(client: any)  {
+    try {
+        let myChannel = client.channels.cache.get(config.priceChannel);
+        let data = await handleGetFromME();
+
+        myChannel.setName(`Floor Price: ◎${Number(data.floorPrice)/1000000000}`);
+        }   catch(error) {
+                console.log(error);
+            }
+
+};
+
+export async function handleTotalVolumeChannel(client: any)  {
+    try {
+        let myChannel = client.channels.cache.get(config.volumeChannel);
+        let data = await handleGetFromME();
+
+        myChannel.setName(`Total Volume: ◎${prettyNumber(data.volumeAll/1000000000)}`);
```

Comment:
We could use `LAMPORTS_PER_SOL` from `@solana/web3.js` here instead https://solana-labs.github.io/solana-web3.js/variables/LAMPORTS_PER_SOL.html